test/unit.test.js

```diff
@@ -34,21 +34,36 @@ describe('matchBrowserLocale', () => {
     const { matchBrowserLocale } = await import('../src/templates/utils-common')
 
     // Both locales match first browser locale - full locale should win.
-    const appLocales = ['en', 'en-US']
+    const appLocales = ['en', 'en-US', 'en-us']
     const browserLocales = ['en-US', 'en']
 
     expect(matchBrowserLocale(appLocales, browserLocales)).toBe('en-US')
+
+    appLocales = ['zh-CN', 'zh-cn']
+    browserLocales = ['zh-CN', 'zh']
```

Comment:
We probably only need one new test that tests that full code is compared case-insensitively.
Eventually, we could also test a case-insensitive match on short (two-letter) locale.
No need to create pointless tests that don't test anything specific.

Reply:
Intended to go as far as I can. Thanks anyway